README.md

```diff
@@ -1,22 +1,67 @@
-RedisMemo
-=========
-Caching is made easy.
+<img align="left" src="https://github.com/chanzuckerberg/redis-memo/blob/main/docs/images/icon.png?raw=true" alt="drawing" width="120"/>
+
+# RedisMemo
 
 [![Gem Version](https://badge.fury.io/rb/redis-memo.svg)](https://badge.fury.io/rb/redis-memo)
 [![Build Status](https://travis-ci.com/chanzuckerberg/redis-memo.svg?branch=main)](https://travis-ci.com/chanzuckerberg/redis-memo)
 [![codecov](https://codecov.io/gh/chanzuckerberg/redis-memo/branch/main/graph/badge.svg?token=XG83PSWPG0)](https://codecov.io/gh/chanzuckerberg/redis-memo)
 
-A Redis-based [version-addressable](https://github.com/chanzuckerberg/redis-memo/wiki/Version-Addressable) caching system. Memoize pure functions, aggregated database queries, and 3rd party API calls.
 
-## Getting Started
-### Add to your Gemfile
+## Add to your Gemfile
 ```ruby
 # -- Gemfile --
 gem 'redis-memo'
 ```
+## Usage
+### Cache simple ActiveRecord queries
+Add two lines to the `User` model:
+```ruby
+class User < ApplicationRecord
+  extend RedisMemo::MemoizeQuery
+  memoize_table_column :id
+end
+```
+
+`SELECT "users".* FROM "users" WHERE "users"."id" = $1` queries will **load the data from Redis** instead of the database:
```

Comment:
```suggestion
Therefore, `SELECT "users".* FROM "users" WHERE "users"."id" = $1` queries will **load the data from Redis** instead of from the database:
```